.github/workflows/docker.yml

```diff
@@ -64,18 +70,18 @@ jobs:
       run: |
         DOCKER_IMAGE=openthread/otbr
         DOCKER_PLATFORMS=linux/amd64,linux/arm/v7,linux/arm64
-        VERSION=latest
+        VERSION=${{ matrix.image_tag }}
 
         TAGS="--tag ${DOCKER_IMAGE}:${VERSION}"
 
         echo ::set-output name=docker_image::${DOCKER_IMAGE}
         echo ::set-output name=version::${VERSION}
-        echo ::set-output name=buildx_args::--platform ${DOCKER_PLATFORMS} \
+        echo ::set-output name=buildx_args::"--platform ${DOCKER_PLATFORMS} \
```

Comment:
Are the quotes necessary here?

Reply:
Yes. It makes sure `--build-arg OTBR_OPTIONS='-DOT_DUA=ON -DOT_MLR=ON'` is correctly set to the `buildx_args` output.